README.adoc

```diff
@@ -217,7 +214,7 @@ include::finish/src/main/java/io/openliberty/guides/microprofile/InventoryManage
 
 === Linking to each available resource
 
-Take a look at your `InventoryManager` and `InventoryUtil` files.
+Take a look at your [file=0]`InventoryManager` and [file=1]`InventoryUtil` files.
```

Comment:
- [ ] file hotspots still not appearing for `InventoryManager` and `InventoryUtil`
    - need to change to `[file=0]` to `[hotspot file=0]`